docs/state-slices.md

```diff
@@ -0,0 +1,84 @@
+# State Slices
+
+Hyperapp uses a single state tree — that is, this single object contains all your application level state and serves as the single source of truth. This also means, if you are coming from Redux/Flux, that you have only one store for each app. A single state tree makes it straightforward to locate a specific piece of state, and allows for incredibly easy debugging.
+
+A single state tree is not free from struggle. It can be daunting to update a part of the state deeply nested in the state tree immutably _and_ without resorting to functional lenses / setters or advanced martial arts.
+
+State slices attempt to address this issue by giving you via actions, a slice of the state tree that corresponds to the namespace where both state and action are declared.
+
```

Comment:
style: State slices address this issue with actions that access a slice of the state tree that corresponds to the namespace where both state and action are declared.